assets/css/education.css

```diff
@@ -1,28 +1,35 @@
 @import url(./style.css);
+@font-face {
+    font-family: "NHaas";
+    src: url(../fonts/NHaas\ Bold.63b9aba.woff2);
+}
 
 /* Dark and Light theme */
```

Comment:
This is a self-explanatory part of the stylesheet. Could you please remove this comment and all spaces before this